CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+v1.8.0-RC1/ 2020-03-05
```

Comment:
add RC2 changelog also.

Reply:
AFAIK if there are no changes then we are not adding.

Reply:
Done As discussed slack thread PTAL.